### PR TITLE
Feature/update concept object to handle date times

### PR DESCRIPTION
**Pull request by THOR300**
# Description

This pull request is an update to the `Concept` objects to assist with the conversion to json from the pydantic object. The issue is the datetime field. 

## Proposed version

Please select the option below that is most relevant from the list below. This
will be used to generate the next tag version name during auto-tagging.

- [ ] Skip auto-tagging
- [X] Patch
- [ ] Minor version
- [ ] Major version

Visit the [Semver website](https://semver.org/#summary) to understand the
difference between `MAJOR`, `MINOR`, and `PATCH` versions.

Notes:

- If none of these options are selected, auto-tagging will fail (integrated soon)
- Where multiple options are selected, the most senior option ticked will be
  used -- e.g. Major > Minor > Patch
- If you are selecting the version in the list above using the textbox, make
  sure your selected option is marked `[x]` with no spaces in between the
  brackets and the `x`

## Type of change

Please select the option(s) below that are most relevant:

- [X] Bug fix
- [ ] New feature
- [ ] Breaking change

## How Has This Been Tested?

Please describe the tests that you added to verify your changes.

## Before submitting

- [X] I've read and followed all steps in the [Making a pull request](https://github.com/climatepolicyradar/cpr-sdk/blob/main/.github/CONTRIBUTING.md#making-a-pull-request)
    section of the `CONTRIBUTING` docs.
- [X] I've updated or added any relevant docstrings following the syntax described in the
    [Writing docstrings](https://github.com/climatepolicyradar/cpr-sdk/blob/main/.github/CONTRIBUTING.md#writing-docstrings) section of the `CONTRIBUTING` docs.
- [X] If this PR fixes a bug, I've added a test that will fail without my fix.
- [X] If this PR adds a new feature, I've added tests that sufficiently cover my new functionality.

### CODE REVIEW

File: tests/test_models.py [hunk 1]
```diff
@@ -640,3 +641,21 @@ def test_vespa_concept_instantiation() -> None:
             model="test_model_2",
             timestamp=datetime.now(),
         )
+
+
+def test_vespa_concept_json_conversion() -> None:
+    """Test that the Vespa concept can be converted to json correctly."""
+    json.dumps(
```
Comment (jesse-c) [inline tests/test_models.py:648]: **suggestion:** Use a fixture or snapshot to assert on the result.

**Review by jesse-c — APPROVED** [review]
Out of curiosity, what was the issue that led to this?

**Comment by THOR300** [comment]
> Out of curiosity, what was the issue that led to this?

It came out when I was trying to update Vespa with a json representation of a Concept. 

If you are trying to convert a concept to json by running `json.dump(Concept.model_dump())` it fails as the date time persists post model_dump

File: tests/test_models.py (hunk 1 above)
Comment (THOR300) [reply tests/test_models.py:648]: I will catch up with you on this, keen to get this in!

**Pull request merged**